units/sdl2_mixer.pas

```diff
@@ -89,6 +89,7 @@ function Mix_Linked_Version: PSDL_Version cdecl; external MIX_LibName {$IFDEF DE
   MIX_INIT_OGG         = $00000010;
   MIX_INIT_MID         = $00000020;
   MIX_INIT_OPUS        = $00000040;
+  MIX_INIT_WAVPACK     = $00000080;
```

Comment:
Should you complete this by adding the typed alias (TMIX_InitFlags)?

Reply:
Done.